### ISSUE TITLE
Badnest stopped working 

### ISSUE THREAD

**Issue by nmsousa76** [therealryanbonham/badnest#15]
Looks like badnest stopped working in hass 2021.6.0.   I also noticed that RedDragons integration if completely gone.

**Comment by bregtvandenbergh** [comment]
Indeed, this is because the manifest.json doesnt containe a version.

Cfr. From the changelog

Version#
For core integrations, this should be omitted.

The version of the integration is required for custom integrations. The version needs to be a valid version recognized by AwesomeVersion like CalVer or SemVer.

**Comment by bregtvandenbergh** [comment]
https://pypi.org/project/awesomeversion/

**Comment by bregtvandenbergh** [comment]
> Looks like badnest stopped working in hass 2021.6.0. I also noticed that RedDragons integration if completely gone.

If you open the manifest.json file of the badnest integration, Just inserts the following:

"version": "1.0.0",

Restart the host and it wil work again. 

Kind regards, 

Bregt

**Comment by DolfLund** [comment]
Thank you for your help ahead of time,

I have tried this and it did not resolve, I am on 2021.6.2

My resulting manifest.json

`{
  "domain": "badnest",
  "name": "Bad Nest (A hack around the Nest component to pull from their internal api)",
  "documentation": "https://custom-components.github.io/badnest",
  "dependencies": [],
  "codeowners": [
    "@USA-RedDragon"
  ],
  "homeassistant": "0.97.0",
  "version": "1.0.0",
  "requirements": []
}
`

Any other suggestions? 


UPDATE: I hadn't rebooted host only HA.

**Comment by bregtvandenbergh** [comment]
For me this worked:

![Screenshot_20210606-221408_Home Assistant](https://user-images.githubusercontent.com/31744864/120938747-baab6480-c714-11eb-9100-c7faf528ae14.jpg)

After reboot host it should work.

Grts

**Comment by bregtvandenbergh** [comment]
Also newest version of ha here

**Comment by DemiMelfice** [comment]
Same issue, home assistant won't allow a basic reset because it knows BadNest wasn't loaded. Should really look at getting this included in an update?
